lib/publiccloud/provider.pm

```diff
@@ -248,6 +249,42 @@ sub destroy_instance {
     $self->ipa(cleanup => 1, running_instance_id => $instance->instance_id);
 }
 
+sub _get_config {
```

Comment:
this '_' before method name isn't it python naming  convention? what does it mean ? just for general understanding

Reply:
I use it to mark a method as private.